python/cudf/cudf/tests/input_output/test_parquet_variant.py

```
@@ -0,0 +1,178 @@
+    ("v_list_struct", _check_in_list_of_struct),
+]
+
+
+@pytest.fixture(scope="module", params=sorted(_NESTED_FIXTURES))
+def nested_variant_df(request, datadir):
+    df = cudf.read_parquet(datadir / _NESTED_FIXTURES[request.param])
+    return request.param, df
+
+
+def test_nested_variant_shape(nested_variant_df):
+    shred_state, df = nested_variant_df
```

Ruff (0.15.12)

[warning] Unpacked variable `shred_state` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)